starter-code/main.js

```diff
@@ -53,6 +53,217 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "5":
+            // 5.- List by name all companies founded in the summer of 2004 (april to june) sorted by date.
+            db.collection('companies').find({$and: [ {founded_year : 2004}, {founded_month: {$in : [4,5,6] } } ] }, { name: 1, _id: 0 })
+              .toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+
+          case "6":
+            // 6.- What companies have offices in "Barcelona".
+            db.collection('companies').find({"offices.city": "Pleasanton"}, { name: 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "7":
+            // 7.- List the 10 companies with more employees sorted ascending (show name and employees).
+            db.collection('companies').find({},
+              { name: 1, number_of_employees:1, _id: 0 })
+              .sort({number_of_employees: -1})
+              .limit(10)
+              //.sort({number_of_employees: -1}) 
+              .toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result.reverse());
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+            break;
+
+          case "8":
+            // 8.- Find the company with the name "Facebook"
+            db.collection('companies').find({ "name": "Facebook" }, { name: 1, _id: 0 }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => { mainMenu() });
+              }
+            })
+          break;
+  
+          case "9":
+            // 9.- How many employees has Facebook?
+            db.collection('companies').find({ "name": "Facebook" }, { name: 1, number_of_employees:1, _id: 0 }).toArray((error, result) => {
+              if (error) {
```

Comment:
Con proyecciones adecuadas no tendrías que modificar tanto el console.log.
```suggestion
            db.collection('companies').find({ "name": "Facebook" }, { number_of_employees:1, _id: 0 }).toArray((error, result) => {
              if (error) {
```

Reply:
Así es!